src/msg/Connection.h

```diff
@@ -255,9 +237,18 @@ struct Connection : public RefCountedObject {
     last_keepalive_ack = t;
   }
 
-};
+protected:
+  Connection(CephContext *cct, Messenger *m)
+    : RefCountedObjectSafe(cct),
+      lock("Connection::lock"),
```

Comment:
You seem to be redoing the lock init which is done inline above; probably some rebase issue?

Reply:
Yes, I'll fix that. Thanks!